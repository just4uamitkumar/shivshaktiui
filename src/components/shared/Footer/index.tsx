import { Stack, Grid } from "@mui/material";
import TypoGraphy from "../../common/TypoGraphy";
import { Link } from "react-router";
import Logo from "../../../styles/assets/images/web/whiteLogo.png";
import SocialLinks from "../SocialLinks";

const Footer: React.FC = () => {
  return (
    <>
      <footer>
        <Grid
          className="footer pt-4 pb-4"
          container
          spacing={2}
          justifyContent={"space-between"}
          alignItems={"center"}
        >
          <Grid className="container" container spacing={4}>
            <Grid size={{ xs: 6, md: 3 }}>
              <Stack className="logo">
                <Link to="/">
                  <img src={Logo} alt="Shiv Shakti" />
                </Link>
              </Stack>
              <Stack>
                <TypoGraphy variant="body1">
                  {
                    "At tincidunt vulputate dis natoque sed parturient proine mperdiet. Quisque lig pede eu, malesuda. Felis. Semper orc gue lorem. Integer ipsum nibh semper suscipit dolor fames interd.."
                  }
                </TypoGraphy>
              </Stack>
            </Grid>
            <Grid size={{ xs: 6, md: 3 }}>
              <Stack>
                <TypoGraphy variant="h4">{"Explore"}</TypoGraphy>
              </Stack>
              <Stack className="footer-links">
                <ul>
                  <li>
                    <Link to="/">About Us</Link>
                  </li>
                  <li>
                    <Link to="/code">Code</Link>
                  </li>
                  <li>
                    <Link to="/develop">Develop</Link>
                  </li>
                  <li>
                    <Link to="/">Contact Us</Link>
                  </li>
                  <li>
                    <Link to="/">My Account</Link>
                  </li>
                  <li>
                    <Link to="/">Sign In</Link>
                  </li>
                </ul>
              </Stack>
            </Grid>
            <Grid size={{ xs: 6, md: 3 }}>
              <Stack>
                <TypoGraphy variant="h4">{"Contact info"}</TypoGraphy>
              </Stack>
              <Stack className="footer-links">
                <ul>
                  <li>
                    <Link to="/devotee">Devotee</Link>
                  </li>
                  <li>
                    <Link to="/">Read</Link>
                  </li>
                  <li>
                    <Link to="/">Listen</Link>
                  </li>
                  <li>
                    <Link to="/">Contact Us</Link>
                  </li>
                  <li>
                    <Link to="/">My Account</Link>
                  </li>
                  <li>
                    <Link to="/">Sign In</Link>
                  </li>
                </ul>
              </Stack>
            </Grid>
            <Grid size={{ xs: 6, md: 3 }}>
              <Stack>
                <TypoGraphy variant="h4">{"Connect with us"}</TypoGraphy>
              </Stack>
              <SocialLinks />
            </Grid>
          </Grid>
        </Grid>
      </footer>
    </>
  );
};

export default Footer;
