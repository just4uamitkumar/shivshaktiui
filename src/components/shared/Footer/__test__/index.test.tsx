import { render } from "@testing-library/react";
import Footer from "../index";
import { describe, expect, it } from "vitest";
import { createMemoryRouter, RouterProvider } from "react-router";

describe("Footer Component", () => {
  it("matches snapshot", () => {
    const router = createMemoryRouter([
      {
        path: "/",
        element: <Footer />,
      },
    ]);

    const footer = render(<RouterProvider router={router} />);
    expect(footer).toMatchSnapshot();
  });

  it("renders the footer with correct content", () => {
    const router = createMemoryRouter([{ path: "/", element: <Footer /> }]);
    const { getByText, getByAltText } = render(
      <RouterProvider router={router} />
    );
    expect(getByText("Explore")).toBeInTheDocument();
    expect(getByText("About Us")).toBeInTheDocument();
    expect(getByText("Contact info")).toBeInTheDocument();
    expect(getByText("Devotee")).toBeInTheDocument();
    expect(getByAltText("Shiv Shakti")).toBeInTheDocument();
  });
});
