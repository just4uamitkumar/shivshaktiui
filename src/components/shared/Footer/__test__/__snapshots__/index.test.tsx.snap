// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Footer Component > matches snapshot 1`] = `
{
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <footer>
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-row MuiGrid-spacing-xs-2 footer pt-4 pb-4 css-1gwgp8s-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-row MuiGrid-spacing-xs-4 container css-1klzo8g-MuiGrid-root"
          >
            <div
              class="MuiGrid-root MuiGrid-direction-xs-row MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-19vcg0t-MuiGrid-root"
            >
              <div
                class="MuiStack-root logo css-nen11g-MuiStack-root"
              >
                <a
                  data-discover="true"
                  href="/"
                >
                  <img
                    alt="Shiv Shakti"
                    src="/src/styles/assets/images/web/whiteLogo.png"
                  />
                </a>
              </div>
              <div
                class="MuiStack-root css-nen11g-MuiStack-root"
              >
                <p
                  class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
                >
                  At tincidunt vulputate dis natoque sed parturient proine mperdiet. Quisque lig pede eu, malesuda. Felis. Semper orc gue lorem. Integer ipsum nibh semper suscipit dolor fames interd..
                </p>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-direction-xs-row MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-19vcg0t-MuiGrid-root"
            >
              <div
                class="MuiStack-root css-nen11g-MuiStack-root"
              >
                <h4
                  class="MuiTypography-root MuiTypography-h4 css-f84ov1-MuiTypography-root"
                >
                  Explore
                </h4>
              </div>
              <div
                class="MuiStack-root footer-links css-nen11g-MuiStack-root"
              >
                <ul>
                  <li>
                    <a
                      data-discover="true"
                      href="/"
                    >
                      About Us
                    </a>
                  </li>
                  <li>
                    <a
                      data-discover="true"
                      href="/code"
                    >
                      Code
                    </a>
                  </li>
                  <li>
                    <a
                      data-discover="true"
                      href="/develop"
                    >
                      Develop
                    </a>
                  </li>
                  <li>
                    <a
                      data-discover="true"
                      href="/"
                    >
                      Contact Us
                    </a>
                  </li>
                  <li>
                    <a
                      data-discover="true"
                      href="/"
                    >
                      My Account
                    </a>
                  </li>
                  <li>
                    <a
                      data-discover="true"
                      href="/"
                    >
                      Sign In
                    </a>
                  </li>
                </ul>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-direction-xs-row MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-19vcg0t-MuiGrid-root"
            >
              <div
                class="MuiStack-root css-nen11g-MuiStack-root"
              >
                <h4
                  class="MuiTypography-root MuiTypography-h4 css-f84ov1-MuiTypography-root"
                >
                  Contact info
                </h4>
              </div>
              <div
                class="MuiStack-root footer-links css-nen11g-MuiStack-root"
              >
                <ul>
                  <li>
                    <a
                      data-discover="true"
                      href="/devotee"
                    >
                      Devotee
                    </a>
                  </li>
                  <li>
                    <a
                      data-discover="true"
                      href="/"
                    >
                      Read
                    </a>
                  </li>
                  <li>
                    <a
                      data-discover="true"
                      href="/"
                    >
                      Listen
                    </a>
                  </li>
                  <li>
                    <a
                      data-discover="true"
                      href="/"
                    >
                      Contact Us
                    </a>
                  </li>
                  <li>
                    <a
                      data-discover="true"
                      href="/"
                    >
                      My Account
                    </a>
                  </li>
                  <li>
                    <a
                      data-discover="true"
                      href="/"
                    >
                      Sign In
                    </a>
                  </li>
                </ul>
              </div>
            </div>
            <div
              class="MuiGrid-root MuiGrid-direction-xs-row MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-19vcg0t-MuiGrid-root"
            >
              <div
                class="MuiStack-root css-nen11g-MuiStack-root"
              >
                <h4
                  class="MuiTypography-root MuiTypography-h4 css-f84ov1-MuiTypography-root"
                >
                  Connect with us
                </h4>
              </div>
              <ul
                class="socialLinks"
              >
                <li>
                  <button
                    class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 320 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
                      />
                    </svg>
                  </button>
                </li>
                <li>
                  <button
                    class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 512 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
                      />
                    </svg>
                  </button>
                </li>
                <li>
                  <button
                    class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 448 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
                      />
                    </svg>
                  </button>
                </li>
                <li>
                  <button
                    class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
                    tabindex="0"
                    type="button"
                  >
                    <svg
                      fill="currentColor"
                      height="1em"
                      stroke="currentColor"
                      stroke-width="0"
                      viewBox="0 0 384 512"
                      width="1em"
                      xmlns="http://www.w3.org/2000/svg"
                    >
                      <path
                        d="M204 6.5C101.4 6.5 0 74.9 0 185.6 0 256 39.6 296 63.6 296c9.9 0 15.6-27.6 15.6-35.4 0-9.3-23.7-29.1-23.7-67.8 0-80.4 61.2-137.4 140.4-137.4 68.1 0 118.5 38.7 118.5 109.8 0 53.1-21.3 152.7-90.3 152.7-24.9 0-46.2-18-46.2-43.8 0-37.8 26.4-74.4 26.4-113.4 0-66.2-93.9-54.2-93.9 25.8 0 16.8 2.1 35.4 9.6 50.7-13.8 59.4-42 147.9-42 209.1 0 18.9 2.7 37.5 4.5 56.4 3.4 3.8 1.7 3.4 6.9 1.5 50.4-69 48.6-82.5 71.4-172.8 12.3 23.4 44.1 36 69.3 36 106.2 0 153.9-103.5 153.9-196.8C384 71.3 298.2 6.5 204 6.5z"
                      />
                    </svg>
                  </button>
                </li>
              </ul>
            </div>
          </div>
        </div>
      </footer>
    </div>
  </body>,
  "container": <div>
    <footer>
      <div
        class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-row MuiGrid-spacing-xs-2 footer pt-4 pb-4 css-1gwgp8s-MuiGrid-root"
      >
        <div
          class="MuiGrid-root MuiGrid-container MuiGrid-direction-xs-row MuiGrid-spacing-xs-4 container css-1klzo8g-MuiGrid-root"
        >
          <div
            class="MuiGrid-root MuiGrid-direction-xs-row MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-19vcg0t-MuiGrid-root"
          >
            <div
              class="MuiStack-root logo css-nen11g-MuiStack-root"
            >
              <a
                data-discover="true"
                href="/"
              >
                <img
                  alt="Shiv Shakti"
                  src="/src/styles/assets/images/web/whiteLogo.png"
                />
              </a>
            </div>
            <div
              class="MuiStack-root css-nen11g-MuiStack-root"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-rizt0-MuiTypography-root"
              >
                At tincidunt vulputate dis natoque sed parturient proine mperdiet. Quisque lig pede eu, malesuda. Felis. Semper orc gue lorem. Integer ipsum nibh semper suscipit dolor fames interd..
              </p>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-direction-xs-row MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-19vcg0t-MuiGrid-root"
          >
            <div
              class="MuiStack-root css-nen11g-MuiStack-root"
            >
              <h4
                class="MuiTypography-root MuiTypography-h4 css-f84ov1-MuiTypography-root"
              >
                Explore
              </h4>
            </div>
            <div
              class="MuiStack-root footer-links css-nen11g-MuiStack-root"
            >
              <ul>
                <li>
                  <a
                    data-discover="true"
                    href="/"
                  >
                    About Us
                  </a>
                </li>
                <li>
                  <a
                    data-discover="true"
                    href="/code"
                  >
                    Code
                  </a>
                </li>
                <li>
                  <a
                    data-discover="true"
                    href="/develop"
                  >
                    Develop
                  </a>
                </li>
                <li>
                  <a
                    data-discover="true"
                    href="/"
                  >
                    Contact Us
                  </a>
                </li>
                <li>
                  <a
                    data-discover="true"
                    href="/"
                  >
                    My Account
                  </a>
                </li>
                <li>
                  <a
                    data-discover="true"
                    href="/"
                  >
                    Sign In
                  </a>
                </li>
              </ul>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-direction-xs-row MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-19vcg0t-MuiGrid-root"
          >
            <div
              class="MuiStack-root css-nen11g-MuiStack-root"
            >
              <h4
                class="MuiTypography-root MuiTypography-h4 css-f84ov1-MuiTypography-root"
              >
                Contact info
              </h4>
            </div>
            <div
              class="MuiStack-root footer-links css-nen11g-MuiStack-root"
            >
              <ul>
                <li>
                  <a
                    data-discover="true"
                    href="/devotee"
                  >
                    Devotee
                  </a>
                </li>
                <li>
                  <a
                    data-discover="true"
                    href="/"
                  >
                    Read
                  </a>
                </li>
                <li>
                  <a
                    data-discover="true"
                    href="/"
                  >
                    Listen
                  </a>
                </li>
                <li>
                  <a
                    data-discover="true"
                    href="/"
                  >
                    Contact Us
                  </a>
                </li>
                <li>
                  <a
                    data-discover="true"
                    href="/"
                  >
                    My Account
                  </a>
                </li>
                <li>
                  <a
                    data-discover="true"
                    href="/"
                  >
                    Sign In
                  </a>
                </li>
              </ul>
            </div>
          </div>
          <div
            class="MuiGrid-root MuiGrid-direction-xs-row MuiGrid-grid-xs-6 MuiGrid-grid-md-3 css-19vcg0t-MuiGrid-root"
          >
            <div
              class="MuiStack-root css-nen11g-MuiStack-root"
            >
              <h4
                class="MuiTypography-root MuiTypography-h4 css-f84ov1-MuiTypography-root"
              >
                Connect with us
              </h4>
            </div>
            <ul
              class="socialLinks"
            >
              <li>
                <button
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 320 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
                    />
                  </svg>
                </button>
              </li>
              <li>
                <button
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 512 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
                    />
                  </svg>
                </button>
              </li>
              <li>
                <button
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 448 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M100.28 448H7.4V148.9h92.88zM53.79 108.1C24.09 108.1 0 83.5 0 53.8a53.79 53.79 0 0 1 107.58 0c0 29.7-24.1 54.3-53.79 54.3zM447.9 448h-92.68V302.4c0-34.7-.7-79.2-48.29-79.2-48.29 0-55.69 37.7-55.69 76.7V448h-92.78V148.9h89.08v40.8h1.3c12.4-23.5 42.69-48.3 87.88-48.3 94 0 111.28 61.9 111.28 142.3V448z"
                    />
                  </svg>
                </button>
              </li>
              <li>
                <button
                  class="MuiButtonBase-root MuiIconButton-root MuiIconButton-sizeMedium css-53g0n7-MuiButtonBase-root-MuiIconButton-root"
                  tabindex="0"
                  type="button"
                >
                  <svg
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 384 512"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M204 6.5C101.4 6.5 0 74.9 0 185.6 0 256 39.6 296 63.6 296c9.9 0 15.6-27.6 15.6-35.4 0-9.3-23.7-29.1-23.7-67.8 0-80.4 61.2-137.4 140.4-137.4 68.1 0 118.5 38.7 118.5 109.8 0 53.1-21.3 152.7-90.3 152.7-24.9 0-46.2-18-46.2-43.8 0-37.8 26.4-74.4 26.4-113.4 0-66.2-93.9-54.2-93.9 25.8 0 16.8 2.1 35.4 9.6 50.7-13.8 59.4-42 147.9-42 209.1 0 18.9 2.7 37.5 4.5 56.4 3.4 3.8 1.7 3.4 6.9 1.5 50.4-69 48.6-82.5 71.4-172.8 12.3 23.4 44.1 36 69.3 36 106.2 0 153.9-103.5 153.9-196.8C384 71.3 298.2 6.5 204 6.5z"
                    />
                  </svg>
                </button>
              </li>
            </ul>
          </div>
        </div>
      </div>
    </footer>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
